src/python/WMCore/Services/Rucio/Rucio.py

```diff
@@ -1096,7 +1102,7 @@ def getDataLockedAndAvailable(self, **kwargs):
         """
         if 'name' not in kwargs:
             raise WMRucioException("A DID name must be provided to the getBlockLockedAndAvailable API")
-        if self.isContainer(kwargs['name']):
+        if self.isContainer(kwargs['name'], scope=kwargs['scope']):
```

Comment:
It might be that there is no `scope` in the kwargs. So we better set a default first.
To avoid typing 'cms' everywhere, maybe we create a global variable in this module called CMS_SCOPE and refer to it whenever needed.

Reply:
That's fine I can change it to `kwargs.get('scope', 'cms')` instead of having a global variable.